lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -375,6 +397,33 @@ class LighthouseReportViewer {
       self.opener.postMessage({opened: true}, '*');
     }
   }
+
+  /**
+   * @param {PSIParams} params
+   */
+  _loadFromPSI(params) {
+    const placeholder = find('.viewer-placeholder-inner', document.body);
+    placeholder.classList.add('lh-loading');
```

Comment:
this blur stuff should probably happen for the getGistFileContentAsJson() path as well.

`_toggleLoadingBlur(bool)`  ?